sev-types/Cargo.toml

```diff
@@ -8,3 +8,4 @@ license = "Apache-2.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+enumerate = { path = "../enumerate" }
```

Comment:
Needs newline. @mbestavros I thought we had a test for this. Am I wrong?

Reply:
I ran into an issue with this as well, CI will complain that `cargo fmt` fails. `cargo fmt` does not want a new line.

Reply:
We do, but... it's somehow not triggering?

We use `tail -c 1` and fail if we see anything, and this file behaves the same way as ones that _did_ fail correctly. Very strange.